user/views.py

```diff
@@ -16,3 +22,69 @@ def form_valid(self, form):
 		login(self.request, user)
 		self.object = user
 		return HttpResponseRedirect(self.get_success_url())
+
+
+class FollowingListView(LoginRequiredMixin, ListView):
+	model = Follow
+	template_name = 'follow/following.html'
+
+	def get_context_data(self, **kwargs):
+		context = super().get_context_data(**kwargs)
+		user = get_object_or_404(User, username=self.kwargs['username'])
+		context['following_list'] = user.following.all()
+		return context
+
+
+class FollowerListView(LoginRequiredMixin, ListView):
+	model = Follow
+	template_name = 'follow/follower.html'
+
+	def get_context_data(self, **kwargs):
+		context = super().get_context_data(**kwargs)
+		user = get_object_or_404(User, username=self.kwargs['username'])
+		context['follower_list'] = user.follower.all()
+		return context
+
+
+class FollowIndexView(LoginRequiredMixin, TemplateView):
+	template_name = 'follow/follow_index.html'
+
+	def get_context_data(self, **kwargs):
+		context = super().get_context_data(**kwargs)
+		login_user = self.request.user
+		target_user = get_object_or_404(User, username=self.kwargs['username'])
+		has_followed = Follow.objects.filter(follow_to=target_user, follow_from=login_user).count() != 0
```

Comment:
plz use [exists()](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#django.db.models.query.QuerySet.exists)
```suggestion
		has_followed = Follow.objects.filter(follow_to=target_user, follow_from=login_user).exists()
```